packages/backend/src/peripherals/database/KeyValueStore.ts

```diff
@@ -37,6 +45,15 @@ export class KeyValueStore extends BaseRepository {
     /* eslint-enable @typescript-eslint/unbound-method */
   }
 
+  async findByKeyWithDefault<K extends KeyValueRecord['key']>(
```

Comment:
well I would extend the findByKey method with fallback arg instead creating new function for it

Reply:
That was my initial implementation, but then it makes two last parameters of `findByKey` optional: `defaultValue` and `trx`. Then you either need to pass `undefined` if you want to use transaction but no default, or the other way around. Both look ugly. That's why I went for a separate method.